src/pages/about.vue

```diff
@@ -3,90 +3,71 @@
     <div class="container">
       <div>
         <h1 class="text-5xl mb-10">
-          {{ $t('about.title') }}
+          {{ $t('about.title', { brand: 'Openverse' }) }}
         </h1>
         <div class="content">
           <p>{{ $t('about.description.content') }}</p>
 
           <i18n path="about.collection.content" tag="p">
             <template #common-crawl>
-              <a
-                :aria-label="$t('about.aria.common-crawl')"
-                href="http://commoncrawl.org/"
-                >{{ $t('about.collection.common-crawl') }}</a
-              >
+              <!-- eslint-disable-next-line @intlify/vue-i18n/no-raw-text -->
+              <a href="http://commoncrawl.org/">Common Crawl</a>
             </template>
           </i18n>
 
           <i18n path="about.planning.content" tag="p">
             <template #meta>
-              <NuxtLink
-                :aria-label="$t('about.aria.meta')"
-                :to="localePath('/meta-search')"
-                >{{ $t('about.planning.meta') }}</NuxtLink
-              >
+              <NuxtLink :to="localePath('/meta-search')">{{
+                $t('about.planning.meta')
+              }}</NuxtLink>
             </template>
             <template #search>
-              <a
-                :aria-label="$t('about.aria.search')"
-                href="https://github.com/wordpress/openverse-frontend/"
-                >{{ $t('about.planning.search') }}</a
+              <!-- eslint-disable @intlify/vue-i18n/no-raw-text -->
```

Comment:
I wonder if to avoid having to disable this error everywhere, we couldn't just create a top-level property on the translation object `"brand": "Openverse"` and then add a translators comment to ignore it?

It's just frustrating disabling eslint errors in the template because you also have to _re-enable_ them or they'll be disabled for the whole rest of the template, which could actually introduce accidental untranslated strings.

Reply:
Good catch! Totally forgot to close the exception comments here. Fixed in [8baeb20](https://github.com/WordPress/openverse-frontend/pull/898/commits/8baeb201b7a975e4b9df59ec6812a5fd552f2f34).

> I wonder if to avoid having to disable this error everywhere, we couldn't just create a top-level property on the translation object "brand": "Openverse" and then add a translators comment to ignore it?

Although it would help to concentrate the brand in one place I don't think this is an acceptable solution as it still leaves the possibility for the brand string will be translated and we will have to verify that the comment is actually honored.